### DISCUSSION TITLE
How to enable debug functions menu

### DISCUSSION THREAD

**Discussion by Ogimo1** [cataclysmbn/Cataclysm-BN#2584] [Q&A]
### Describe the bug

When I open the debug menu to be a big ol cheater (and testing new things) there are no options except "info"

### Steps To Reproduce

1. Open debug menu
2. Be sad

### Screenshots

![asd](https://user-images.githubusercontent.com/129221712/230600616-36d83521-4301-4bab-963e-ef7d1a551705.PNG)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: 09f67f8 [64-bit]
- Graphics Vers

**Comment by olanti-p** [comment]
You have to bind a key to debug menu in keybindings, otherwise it's "hidden"

**Reply by Ogimo1** [reply]
I do have the debug menu set as a keybind, for example mine is set to "?". However when I use that keybind it still only pops up the debug function window with just the info option.

**Reply by olanti-p** [reply]
"?" is a bad choice, it's a reserved keybind for keybindings window.
Try tilde or one of the F1-F12 keys.
Also, make sure you have set a keybind to debug menu, and not debug mode.

**Reply by Ogimo1** [reply]
I rebound the key to "~" as suggested and it worked just fine! I guess having it bound to "?" created some issues. Thanks for your help.
